Assisments/Assignment1/src/Cart.java

```diff
@@ -0,0 +1,122 @@
+import java.util.HashMap;
+import java.util.Set;
+
+/*
+ *  This Class hold the data members and methods.
+ *  cartItems is HashMap which stores the  object of Items class and the Quantity of Item
+ *  bill is double type variable used to save the total bill
+ *  
+ * */
+public class Cart {
+	
+		HashMap<Items, Integer> cartItems = new HashMap<Items, Integer>();
+		double bill;
+		/*
+		 * Constructor to initialize the object
+		 * */
+		public Cart() {
+			
+			bill=0.0;
+		}
+		/*
+		 * Constructor to initialize the object 
+		 * @param item is Object of class items contain name and price of item
+		 * @param quantity represents the quantity of item taken
+		 * */
+		public Cart(Items item, Integer quantity){
+			cartItems.put(item, quantity);
+			bill=0.0;
```

Comment:
When a user enter new item, then previous added bill amount will be zero.

Reply:
Sir, I just walk through the code, it will not cause the error as in file ShoppingCart.java  line 96,
I am calling addItem to add new item in list. The Cart constructor will only be called when the new user arrives. 
Although I will remove the bill variable from the constructor and put it in the new function for bill calculation